test/hotspot/jtreg/serviceability/jvmti/vthread/SelfSuspendDisablerTest/SelfSuspendDisablerTest.java

```diff
@@ -66,10 +66,27 @@ private static void sleep(long millis) {
     private static void testJvmtiThreadState(Thread thread, int expectedState) {
         String kindStr = thread.isVirtual()? "virtual " : "platform";
         int state = getThreadState(thread);
+        boolean pass = true;
 
         System.out.printf("Expected %s thread state: %06X got: %06X\n",
                           kindStr, expectedState, state);
-        if ((state & expectedState) != expectedState) {
+        if (expectedState == SUSPENDED) {
+            if ((state & expectedState) != expectedState) {
+                /* Fail if the required bits are not set. */
+                pass = false;
+            }
+	    /* SUSPENDED_OPTIONAL = JVMTI_THREAD_STATE_INTERRUPTED | JVMTI_THREAD_STATE_IN_NATIVE */
```

Comment:
Please correct the indentation of this line (use only spaces).

Reply:
Updated.